package.json

```diff
@@ -213,7 +213,7 @@
     "prepush": "npm run -s install-if-no-packages && node bin/pre-push-hook.js",
     "rm": "node bin/rm.js",
     "postshrinkwrap": "node bin/trim-shrinkwrap.js && touch node_modules",
-    "prestart": "npm run -s install-if-deps-outdated && node bin/welcome.js && ( check-node-version --package || exit 0 )",
+    "prestart": "npm run -s install-if-deps-outdated && node bin/welcome.js && check-node-version --package",
```

Comment:
This will stop building if we don't match exactly. Ideally we only fail when major versions don't match
```js
check-node-version --node 8 --npm 5
```

but I think we should also have a loud warning if we don't match exact engine warnings, as we'll run into trouble if folks shrinkwrap with the wrong versions.

Reply:
Ah right, I just felt bad doing that because it was another place to update major version numbers (`.nvmrc`, `.npmrc`, engines, and now here). I could hadcode them in that way but before doing that I’ll see if I can do some wrangling to get just the major version from engines or the rc scripts.